### PR TITLE
SID change triggered MAC address generation

### PR DESCRIPTION
**Pull request by LostFuse**
The main problem was that MAC persisted after moving the SD card to another board.
Instead of regenerating the MAC on each boot, it is now triggered by the Serial ID change.

### CODE REVIEW

File: docs/rte-mac-setup.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 
 After flashing the card with the new image, the `MAC` will be generated from the
-board's serial id and stored in `U-Boot`'s environment variable on the first
+board's `Serial ID` and stored in `U-Boot`'s environment variable on the first
 boot.
 
-You can change the address for a single boot by modifying the variable from
+If no `Serial ID` is found in U-Boot environment, tha `MAC` will be regenerated.
+
+If `Serial ID` change, the `MAC` will be regenerated again.
```
Comment (tomzy-0) [inline docs/rte-mac-setup.md]: @leckijakub please explain here how the Serial ID can be changed (same SD card, different device)

**Comment by tomzy-0** [comment]
@leckijakub also please fill up the changelog in this PR

File: docs/rte-mac-setup.md (hunk 1 above)
Comment (LostFuse) [reply docs/rte-mac-setup.md]: @TomaszAIR explanation added.

**Comment by LostFuse** [comment]
> @leckijakub also please fill up the changelog in this PR

@TomaszAIR I updated the changelog to version `v0.7.1`

**Pull request merged**